packages/block-library/src/latest-posts/edit.js

```diff
@@ -148,6 +151,20 @@ export default function LatestPostsEdit( { attributes, setAttributes } ) {
 		]
 	);
 
+	// If a user clicks to a link prevent redirection and show a warning.
+	const { createWarningNotice, removeNotice } = useDispatch( noticeStore );
+	let noticeId;
+	const showRedirectionPreventedNotice = ( event ) => {
+		event.preventDefault();
+		// Remove previous warning if any, to show one at a time per block.
+		removeNotice( noticeId );
+		noticeId = `block-library/core/latest-posts/redirection-prevented/${ instanceId }`;
+		createWarningNotice( __( 'Links are disabled in the editor.' ), {
+			id: noticeId,
```

Comment:
suggestion (non-blocking): I think you'll get similar results if you use `block-library/core/latest-posts/redirection-prevented` as an ID and let the reducer handle duplicates.

https://github.com/WordPress/gutenberg/blob/9e2c238d1bda8aa838e242a840e4984240ff54db/packages/notices/src/store/reducer.js#L23-L27

## Screencast
https://user-images.githubusercontent.com/240569/167081066-1e3b6099-6ccd-450a-ae94-ebf13933fc4b.mp4

Reply:
I saw that, yes and while I'm still not 100% sure, I added the `removeNotice` because even if it doesn't seem too possible, if in the future that store implementation changed, we would end up with multiple notifications.. WDYT?

Reply:
I agree. This way, it's more future-proof.